docs/jwst/pathloss/description.rst

```diff
@@ -6,23 +6,30 @@ Overview
 --------
 
 The pathloss correction step calculates the correction to apply to spectra
-when the 1-d extraction is performed.  This correction accounts for losses
+when the 1-d extraction is performed.  The motivation behind the correction
+is different for NIRSPEC and NIRISS, while that for MIRI has not been
+implemented in Build 7.3.  For NIRSPEC, this correction accounts for losses
 in the optical system due to light being scattered outside the grating, and
-to light not passing through the aperture.
+to light not passing through the aperture, while for NIRISS_SOSS data it
+corrects for the flux that falls outside the subarray.
 
 Background
 __________
 
 The correction is applicable to NIRSPEC IFU, MSA and FIXEDSLIT exposure types,
-to NIRISS SOSS data, and to MIRI LRS and MRS data, although the MIRI and NIRISS
-corrections are not implemented in Build 7.
-The description of how the reference files were created and how they are to be
+to NIRISS SOSS data, and to MIRI LRS and MRS data, although the MIRI
+correction has not been implemented prior to Build 7.3.
```

Comment:
Do we know if something for MIRI will be coming for B7.3? Perhaps we shouldn't mention build numbers at all and just say it hasn't been implemented yet (period).

Reply:
I haven't heard anything about MIRI pathloss plans.